climada/util/plot.py

```diff
@@ -470,6 +470,17 @@ def geo_im_from_array(
             cmap=cmap,
             **kwargs,
         )
+        # handle NaNs in griddata
+        color_nan = "gainsboro"
+        if np.any(np.isnan(grid_im)):
```

Comment:
A bit of mostly pointless optimization ^^
```suggestion
        if np.any(np.isnan(x) for x in grid_im):
```